.github/workflows/ci.yml

```diff
@@ -27,6 +31,27 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Git config
+        run: git config --global --add safe.directory /__w/azionapi-openapi/azionapi-openapi
+        shell: bash
+
+      - name: install openapi-linter
+        run: npm i -g @superfaceai/openapi-linter
+
+     #get all .yaml files which were added/modified
+      - name: Get changed files
+        id: changed-files
+        uses: tj-actions/changed-files@v35
+        with:
+          files: |
+            **/*.yaml
+
+      - name: Run linter for changed files
+        run: |
+          for file in ${{ steps.changed-files.outputs.all_changed_files }}; do
```

Comment:
I prefer to scan all the files, because there are few of them and by scanning all the files we guarantee that the whole project is always fine. The only way to break this is with a forced merge with admin permissions.

Reply:
Scanning all files every time makes not much sense to me. Checking every modified file is time and cost-efficient, either way, forced merges should not be done anyway.